### PR TITLE
Made  "arrow" on dropdown for GCI submenu showing to the ">" right 

### PR DESCRIPTION
**Pull request by Dharmesh-Poddar**
#461 
screenshot from 2018-12-26 12-56-07

### CODE REVIEW

**Comment by Dharmesh-Poddar** [comment]
@abishekvashok  please review..

**Comment by plxity** [comment]
LGTM 👍

**Comment by kksinghal** [comment]
Please provide a preview link

**Comment by Dharmesh-Poddar** [comment]
@KSNINJA https://loving-mahavira-1fd963.netlify.com

File: css/custom.css [hunk 1]
```diff
@@ -262,16 +262,17 @@ nav{
   margin-left: 250px;
   margin-top: 350px;
 }
-.has-submenu{
+ .has-submenu{
   position: relative;
-}
+}
```
Comment (abishekvashok) [inline css/custom.css:267]: Please remove this space

File: css/custom.css [hunk 2]
```diff
@@ -262,16 +262,17 @@ nav{
   margin-left: 250px;
   margin-top: 350px;
 }
-.has-submenu{
+ .has-submenu{
```
Comment (abishekvashok) [inline css/custom.css:265]: Please remove this space

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Pull request merged**